Mathlib/MeasureTheory/Integral/Lebesgue/Basic.lean

```diff
@@ -272,6 +272,54 @@ theorem setLIntegral_congr_fun {f g : α → ℝ≥0∞} {s : Set α} (hs : Meas
   rw [EventuallyEq]
   rwa [ae_restrict_iff' hs]
 
+section
+
+@[simp]
+theorem lintegral_eq_zero_iff' {f : α → ℝ≥0∞} (hf : AEMeasurable f μ) :
+    ∫⁻ a, f a ∂μ = 0 ↔ f =ᵐ[μ] 0 := by
+  refine ⟨fun h ↦ ?_, fun h ↦ (lintegral_congr_ae h).trans lintegral_zero⟩
```

Comment:
Could you add a comment to the proof saying that this proof implicitely uses Markov inequality, but that for import reasons you are instead doing it by hand?

Reply:
All done.